build_debian.sh

```diff
@@ -28,6 +28,10 @@
 ## Enable debug output for script
 set -x -e
 
+CONFIGURED_ARCH=$([ -f .arch ] && cat .arch || echo $HOST_ARCH)
```

Comment:
>$HOST_ARCH [](start = 52, length = 10)

sonic_arch has nothing to do with host_arch, suggest set amd64 as the default if .arch is missing

Reply:
Ok, will do this

Reply:
Done